manifest-v2.json

```diff
@@ -126,6 +126,14 @@
       "packageType": "Zip",
       "useCaseName": "Hello World Example"
     },
+    {
+      "directory": "dotnet/hello-pt",
```

Comment:
directory should be `dotnet6/hello-pt` right?

Reply:
absolutely! I thought this would've caught in tests.. thanks for spotting it!!